### PR TITLE
Perform selector clicks and booking clicks serially

### PR DESCRIPTION
**Pull request by sushrut111**
- Added synchonization

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -351,6 +358,11 @@ if (window.location.hash) {
 var current_href = location.href;
 setInterval(function () {
   if (current_href !== location.href) {
+    while(alreadySetIntervalsForEnableRefresh.length>0){
```
Comment (prayuj) [inline contentScript.js:361]: @sushrut111 Did you notice multiple intervals existing?

Comment (sushrut111) [reply contentScript.js:361]: Yes I did.
But the line you are commenting on is to deregister all intervals on page change.

File: contentScript.js [hunk 2]
```diff
@@ -154,12 +162,15 @@ const repFun = () => {
   //   if(!!!allow_multiple) $('.register-btn').trigger('click');
   // })
 
-  const dispatchSelectorClick = () => {
+  const dispatchSelectorClick = async() => {
+    await sleep(500);
     for (let index = 0; index < checked_buttons.length; index++) {
       const element = checked_buttons[index];
-      setTimeout(() => {
+      await sleep(5);
```
[the remark this answers is not in the document]
Comment (sushrut111) [reply contentScript.js:169]: Yes. There isn't any point of having unnecessary delay.

File: contentScript.js (hunk 1 above)
Comment (prayuj) [reply contentScript.js:361]: Yup, that's why I asked if you noticed multiple and that's why you were popping them

**Comment by prayuj** [comment]
@sushrut111 Looks good to merge 👍

**Comment by sushrut111** [comment]
I have already release a pre release version v3.6.1
I am thinking of waiting till evening to see if people face issues - then we can merge.

**Pull request merged**